match/new_orleans_pd.py

```diff
@@ -1,7 +1,7 @@
 from lib.date import combine_date_columns
 from lib.path import data_file_path, ensure_data_dir
 from datamatch import (
-    ThresholdMatcher, JaroWinklerSimilarity, DateSimilarity, ColumnsIndex
+    ThresholdMatcher, JaroWinklerSimilarity, DateSimilarity, ColumnsIndex, NoopIndex, Swap
```

Comment:
`Swap` seems to be an extraneous import?

Reply:
Thanks.